helm-charts/reference-server/values.yaml

```diff
@@ -9,6 +9,7 @@ container:
 
 config:
   app:
+    horizonEndpoint: https://horizon-testnet.stellar.org
```

Comment:
Where is the `passphrase` set for `public` or `testnet`? 

It would be nice to have the consistent pattern with https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/resources/config/anchor-config-default-values.yaml#L34

Reply:
Oh, I see. It is the way the reference server is configured. I will approve this PR.